common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -17,23 +17,27 @@
 import org.opensearch.ml.common.connector.Connector;
 import org.opensearch.ml.common.model.MLModelConfig;
 import org.opensearch.ml.common.model.TextEmbeddingModelConfig;
+import org.opensearch.ml.common.transport.connector.MLCreateConnectorInput;
 
 import java.io.IOException;
-import java.util.Map;
+import java.time.Instant;
 
 import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
-import static org.opensearch.ml.common.connector.Connector.createConnector;
 
 @Data
 public class MLUpdateModelInput implements ToXContentObject, Writeable {
     
-    public static final String MODEL_ID_FIELD = "model_id"; // mandatory
+    public static final String MODEL_ID_FIELD = "model_id"; // passively set when passing url to rest API
     public static final String DESCRIPTION_FIELD = "description"; // optional
-    public static final String MODEL_VERSION_FIELD = "model_version"; // optional
+    public static final String MODEL_VERSION_FIELD = "model_version"; // passively set when register model to a new model group
     public static final String MODEL_NAME_FIELD = "name"; // optional
     public static final String MODEL_GROUP_ID_FIELD = "model_group_id"; // optional
     public static final String MODEL_CONFIG_FIELD = "model_config"; // optional
+    public static final String CONNECTOR_FIELD = "connector"; // optional
     public static final String CONNECTOR_ID_FIELD = "connector_id"; // optional
+    // The field CONNECTOR_UPDATE_CONTENT_FIELD need to be declared because the update of Connector class relies on the MLCreateConnectorInput class
```

Comment:
It’s hard to explain in a few sentences or so, but the idea here is we need a separate field for connector update because the class we create and update connector is different from the connector class. However for the clarity and maintenance purpose we need to keep the connector_field to be strictly connector class, so we need a separate field named CONNECTOR_UPDATE_CONTENT_FIELD